doc/api_methods.md

```diff
@@ -70,14 +70,14 @@ Returns an array of layers in the map. The first is the base layer.
 
 ### vis.addOverlay(_options_)
 
-Adds an overlay to the map that can be either a zoom control, a tooltip or an infobox.
+Adds an overlay to the map that can be either a zoom control, a tooltip or an infowindow (hover action).
 
 #### Arguments
 
 Option | Description
 --- | ---
 layer | layer from the visualization where the overlay should be applied (optional)
-type | zoom / tooltip / infobox
+type | zoom / tooltip / [infowindow with hover action](#visaddinfowindowmap-layer-fields--options)
```

Comment:
These are actually the string literals that you can set as types. `infobox` is actually one of them, so I'd live it the way it was.